App/Sagas/LocationSagas.js

```diff
@@ -0,0 +1,17 @@
+import { call, put } from 'redux-saga/effects'
+import { delay } from 'redux-saga'
+import LocationActions from '../Redux/LocationRedux'
+
+// 15 minutes
+const updateDelay = 15 * 60 * 1000
+
+export function* getNearbyUpdates (api, action) {
+  let response = yield call(api.getNearby)
```

Comment:
Not a huge deal since it's only 2 lines, but this code is duplicated down in the endless loop too.  Two other options:

1. set a flag to `false` then go into the while block... only do the call to delay when the flag is `true`... then the next line of code, set the flag to `true`.
1. extract these two lines to a `async` function and do a `call` to it instead.

My preference would be for #2 because it'll be simple to test.

This is super nitpicky though.  Just throwing it out there.  You probably have considered it already.  Since my face is up for reviewing this, I thought I'd say it.  =)